### DISCUSSION TITLE
How to get rid of status bar on Android native app on auto-rotate

### DISCUSSION THREAD

**Discussion by chakri2907** [videojs/video.js#8363] [Q&A]
We have integrated video-js to support landscape mode for the videos. But when we go to landscape mode using enterFullWindow() on auto-rotate, we see status bar at the top.
Steps are:
Click on Watch in portrait mode.
Auto-rotate the device to go to landscape mode
Programmatically used enterFullWidow() method to change the orientation, video changes correctly and plays in landscape mode, however we see status bar at the top. 

We see this issue in android native app (which is wrapper app).
In IOS app, we dont see this issue and there we are using perferFullWindow.

Any suggestions ??

**Comment by chakri2907** [comment]
@heff @brainopia @nbibler @rafaelgaspar Any suggestions on the query.
Apologies for tagging you, was in desperate to get the RCA of the issue

**Comment by mister-ben** [comment]
`enterFullWidow()` makes the player fit the window. Whether or not a status bar is displayed is up to the browser/app, it can't be hidden with javascript. `requestFullscreen()` would enter actual fullscreen mode.

**Comment by chakri2907** [comment]
Thanks for the response.

We have requirement of adding a overlay panel with some text and some click actions.
And on using  request full-screen method, it is not working and we r  not getting control over the overlay panel.

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by chakri2907** [reply]
@mister-ben any suggestions plz.  We tried with "request full-screen" method
If we click on fullscreen option, video turns to landscape mode and full window is used.
If we rotate without any click action, video turns to landscape mode but we see above blue bar (safe bar). This is happening in Android only.
